src/Identity/UI/src/build/IdentityUI.targets

```diff
@@ -0,0 +1,23 @@
+<Project>
```

Comment:
This file needs to be named `Microsoft.AspNetCore.Identity.UI.targets` otherwise msbuild won't import it